training/evaluate.py

```diff
@@ -439,7 +439,7 @@ def check_valid_combination(dataset, feature, domain):
     is_valid = check_valid_combination(args.dataset, args.feature, args.domain)
 
     if is_valid:
-        option_manager = BaseOptions(args.model, args.dataset, args.feature, args.domain)
+        option_manager = BaseOptions(args.model, args.dataset, args.feature, False, args.domain)
```

Comment:
Could you remove magic number of False? Instead, please insert variable X = False and then use it here for readability?